pkg/porter/cnab.go

```diff
@@ -131,8 +154,12 @@ func (o *sharedOptions) validateInstallationName(args []string) error {
 // defaultBundleFiles defaults the porter manifest and the bundle.json files.
 func (o *bundleFileOptions) defaultBundleFiles(cxt *context.Context) error {
 	if o.File != "" { // --file
-		bundleDir := filepath.Dir(o.File)
-		o.CNABFile = filepath.Join(bundleDir, build.LOCAL_BUNDLE)
+		// If o.Dir not set, assume the build context dir is in the same
```

Comment:
Oh wait, this doesn't line up with what we discussed. I thought we were going to use the same semantics as docker build, which defaults the build context to the current working directory when unspecified.

So if you run `porter build -f porter/porter.yaml` and the build context is set to `.`.

If we are consistent with how docker does it, then we can set the default value of o.Dir to `"."` and remove a bit of logic in this file, no?

Reply:
Oops, indeed.  I was fooled/forgot that discussion when I looked at the previous behavior.  It should all be updated now to reflect what you mentioned above.  Thank you.